Header header
int32 frame_number
int32 player_id
int32 play_number
int32 amount
